bigquery.json

```diff
@@ -0,0 +1 @@
+{"type": "service_account", "project_id": "test"}
```

Comment:
This file is generated by some storage connector test. We should put it in the gitignore.

Reply:
Done